### PR TITLE
Rule Botannia: British Bots for the Restaurant

### PR DESCRIPTION
**Pull request by EOBGames**
## About The Pull Request
Adds British Bots to the restaurant. Their imperial tastes span a vast selection of items, so be sure you're able to accomodate them!
britbots
Also, adds some food items specifically for the Britbots: Beef Wellington, Indian Chicken Curry, and the esteemed Full English breakfast.
Finally, adds a new telescreen type for the bar- useful for keeping tabs on your customers from the comfort of the kitchen. Will require mapping changes, which I'm considering as out of scope for this PR.
The directionals for the bots, and the Bobby outfit, were both made by @tralezab.
The sound was made by @Qustinnus.
In addition, this includes the changes from #57541.

## Why It's Good For The Game
Variety is the spice of life, and that goes double for the kitchen.

## Changelog
:cl:Inept, Tralezab, Qustinnus
add: The Robot Earth Tourism Program has expanded, bringing in the polite British-Bots, who are hungry for some food that reminds them of the Empire. Bot Appetit!
/:cl:

### CODE REVIEW

**Comment by Pugie** [comment]
I hope nobody asks me for a bloody cup o' tea...

File: code/modules/food_and_drinks/restaurant/customers/_customer.dm [hunk 1]
```diff
@@ -142,3 +142,20 @@
...
 	wait_for_food_lines = list("Ay ay ay, what's taking so long...", "Are you ready yet, amigo?")
+
+/datum/customer_data/british
+	nationality = "Space-British"
+	base_icon = "british"
+	prefix_file = "strings/names/british_prefix.txt"
+	speech_sound = 'sound/creatures/tourist/tourist_talk_british.ogg'
+	clothing_sets = list("british_gentleman", "british_bobby")
+	orderable_objects = list(
+	/datum/venue/restaurant = list(/obj/item/food/fishandchips = 10, /obj/item/food/soup/stew = 10, , /obj/item/food/salad/ricepudding = 5, /obj/item/food/grilledcheese = 5, /obj/item/food/pie/meatpie = 5, /obj/item/food/benedict = 5, /obj/item/food/full_english = 2, /obj/item/food/soup/indian_curry = 3, /obj/item/food/beef_wellington/slice = 2),
```
Comment (Iamgoofball) [inline code/modules/food_and_drinks/restaurant/customers/_customer.dm:153]: remove the curry from this list, it's a known fact the british don't spice their food

**Comment by Paxilmaniac** [comment]
I think you spelled it wrong, it's actually supposed to be "Bri'ish"

**Review by Twaticus — APPROVED** [review]

**Comment by MrMelbert** [comment]
image
image

manuel testmerge, round 157895, british bots appeared to have some wonky clothing.

**Comment by Qustinnus** [comment]
lmao they got amerifat clothing

File: code/modules/food_and_drinks/restaurant/_venue.dm [hunk 2]
```diff
@@ -5,7 +5,7 @@
 	///Max amount of guests at any time
 	var/max_guests = 6
 	///Weighted list of customer types
-	var/list/customer_types = list(/datum/customer_data/american = 5, /datum/customer_data/italian = 3, /datum/customer_data/french = 3, /datum/customer_data/japanese = 3, /datum/customer_data/japanese/salaryman = 2, /datum/customer_data/mexican = 10)
+	var/list/customer_types = list(/datum/customer_data/american = 5, /datum/customer_data/italian = 3, /datum/customer_data/french = 3, /datum/customer_data/japanese = 3, /datum/customer_data/japanese/salaryman = 2, /datum/customer_data/mexican = 3, /datum/customer_data/british = 4)
```
Comment (Qustinnus) [inline code/modules/food_and_drinks/restaurant/_venue.dm:8]:
```suggestion
	var/list/customer_types = list(/datum/customer_data/american = 5, /datum/customer_data/italian = 3, /datum/customer_data/french = 3, /datum/customer_data/japanese = 3, /datum/customer_data/japanese/salaryman = 2, /datum/customer_data/mexican = 3, /datum/customer_data/british/bobby = 2, /datum/customer_data/british/gent = 2)
```

**Comment by adamkad1** [comment]
apparently some things dont pay up

**Pull request merged**